### PR TITLE
Added benchmarks for kafka channel/kafka broker

### LINKED ISSUE #573: Test KafkaChannel throughput and latency
**Problem**
Similar (or identical) to the throughput and latency tests for InMemoryChannel (https://github.com/knative/eventing/issues/1804), we should test KafkaChannel throughput and latency.

**Exit Criteria**
Test(s) can be run manually to submit throughput and latency metrics to Mako.

**Time Estimate (optional):** 

**Additional context (optional)**

/area performance

@slinkydeveloper

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Fixes #573

### CODE REVIEW

File: test/performance/broker-kafka/100-broker-perf-setup.yaml [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Broker
+metadata:
+  name: kafka-test-broker
+  namespace: perf-eventing
+spec:
+  channelTemplateSpec:
+    apiVersion: messaging.knative.dev/v1alpha1
+    kind: KafkaChannel
+
```
Comment (matzew) [inline test/performance/broker-kafka/100-broker-perf-setup.yaml]: we want to capture _specific_ for the channel ? e.g `partitions`  ?  sure, could be added later!

Comment (slinkydeveloper) [reply test/performance/broker-kafka/100-broker-perf-setup.yaml]: I think the purpose of these scripts are to serve as a basis to run the tests. Then, based on what you are searching, you tune it :)

Comment (matzew) [reply test/performance/broker-kafka/100-broker-perf-setup.yaml]: perfect!

**Pull request merged**